shared/domains/chelonia/GIMessage.js

```diff
@@ -72,6 +72,27 @@ export class GIMessage {
     })
   }
 
+  // GIMessage.assign is necessary when make an GIMessage object having the same identity id()
+  // https://github.com/okTurtles/group-income/issues/1503
+  static assign (
```

Comment:
I recommend renaming this to `clone` or `cloneWith`

Reply:
I used the name `assign` from the `Object.assign`. We can use `cloneWith` either.